R-package/R/lgb.train.R

```diff
@@ -47,7 +47,7 @@
 #' }
 #' @export
 lgb.train <- function(params = list(),
-                      data,
+                      data = data,
```

Comment:
```suggestion
                      data,
```

Can you please revert this? This one is not appropriate, since it would be getting an object called `data` from the calling environment. No object like that is guaranteed to exist.